### DISCUSSION TITLE
bug if chatid>integer (2147483647)

### DISCUSSION THREAD

**Discussion by RepkaPipetka** [aso14/DTelegram#8] [General]
userchatid:=r.result[0].callback_query.from.id;
result -> 0
JSON:
"message":{
     "message_id":1032,
     "from":{
           "id":5163344795,
           "is_bot":false,
           "first_name":"deleted",
           "language_code":"ru"},
     "chat":{
           "id":5163344795,
           "first_name":"deleted",
           "type":"private"},
     "date":1726750683,
     "text":"start"}}

**Comment by aso14** [comment]
Thanks !
